Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMajor(from: "1.2.2")),
-        .package(url: "https://github.com/apple/swift-package-manager.git", branch: "main"),
+        .package(url: "https://github.com/apple/swift-package-manager.git", revision: "82e3974c7714783bab1da11e29d053422fdfba9a"), // 5.10 Release
```

Comment:
I created a new branch `5.10` in this repo that points to SwiftPM's `release/5.10` branch. Let's keep the `branch: "main"` here.